### PR TITLE
Add Signed In Navigation

### PR DESCRIPTION
**Pull request by davidrapson**
Adds signed in status and account navigation to the header.

screen shot 2015-10-05 at 14 24 41

**Header before**
screen shot 2015-10-05 at 12 58 49

**Header after**
screen shot 2015-10-05 at 12 58 54

### CODE REVIEW

**Comment by rtyley** [comment]
Could @joelochlann or @chrisjowen review this?

**Comment by afiore** [comment]
:+1: Looking good

File: app/configuration/Links.scala [hunk 1]
```diff
@@ -1,6 +1,10 @@
 package configuration
 
-case class Links(href: String, title: String)
+import utils.StringUtils._
+
+case class Links(href: String, title: String) {
```
Comment (afiore) [inline app/configuration/Links.scala]: Just one naming pedantry... Can we rename `Links` to `Link`?

**Comment by davidrapson** [comment]
Confirmed on PROD

screen shot 2015-10-09 at 12 18 22

**Pull request merged**